web/source/keymanweb.js

```diff
@@ -3525,22 +3535,106 @@
     // IE: call _SelectionChange when the user changes the selection 
     if(document.selection  &&  !keymanweb.legacy)
       util.attachDOMEvent(document, 'selectionchange', keymanweb._SelectionChange);
-       
-    // Add event listeners and attach manually-positioned KMW objects
-    if(keymanweb.options['attachType'] != 'manual')  // I1961
```

Comment:
I'll admit I haven't directly checked for that... I thought it was a flag automatically set depending upon the OS.  If I recall correctly, the _only_ thing triggered by this was the IE-specific stuff that was recently removed... though on a relook it seems that MutationObservers ought only be activated when it's not 'manual.'  Looks like a good catch, though I'm surprised it took this long to be spotted.